packages/cli/src/endo.js

```diff
@@ -277,6 +280,30 @@ export const main = async rawArgs => {
       }
     });
 
+  program
+    .command('bundle <application-path>')
```

Comment:
What's an application path? Perhaps I know enough to guess that it's the root module path that I would have given to `bundleSource()`, but how about the hapless `endo bundle --help` user?

Does `command` support extended descriptions with examples and such?

I'd like to see a short example, something like:

> Given `app1.js` that imports `mod1.js`, you can bundle them with `endo bundle index.js -n app1`.

Then... what object does `app1` get bound to? Again, I can perhaps guess that it's the module namespace exported by `app1.js`, but we should share that info with `endo bundle --help` users.

Reply:
Yeah, `commander` does support extended usage. I need to do a `--help` documentation pass over all of them.